shallow_backup/cli.py

```diff
@@ -60,6 +60,7 @@ def cli(add, rm, show, all, dotfiles, configs, packages, fonts, old_path, new_pa
 			print_red_bold("Removed config file...")
 		elif destroy_backup:
 			backup_home_path = get_config()["backup_path"]
+			backup_home_path = os.path.abspath(os.path.expanduser(backup_home_path))
```

Comment:
Let's extract `os.path.abspath(os.path.expanduser(VAR))` to a function that lives in `utils.py` to reduce code repetition.